src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -4,6 +4,8 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.Collections.Generic;
```

Comment:
Please remove, we generally autogenerate these files and the format requires we use fully qualified names.